.devcontainer/docker-compose.yml

```diff
@@ -68,17 +68,21 @@ services:
         restart: always
         command: start-dev --import-realm
         environment:
-            KC_HOSTNAME: keycloak
+            KC_HOSTNAME: keycloak.local
             KC_HOSTNAME_PORT: ${KEYCLOAK_PORT:-8080}
             KC_HTTP_PORT: ${KEYCLOAK_PORT:-8080}
             KEYCLOAK_ADMIN: admin
             KEYCLOAK_ADMIN_PASSWORD: _mywWorld_
             KC_PROXY: edge
-            IQGEO_DOMAIN: http://localhost:${APP_PORT:-80}
+            IQGEO_DOMAIN: http://Mikes-MacBook-Pro.local:${APP_PORT:-80}
```

Comment:
is a `.local` here necessary?

Reply:
maybe should be an ENV variable that defaults to localhost?

Reply:
This was left in by mistake. it has been removed.